packages/makeup-navigation-emitter/src/index.js

```diff
@@ -1,77 +1,186 @@
 import * as KeyEmitter from 'makeup-key-emitter';
 import * as ExitEmitter from 'makeup-exit-emitter';
 
-const dataSetKey = 'data-makeup-index';
-
-// todo: autoInit: -1, autoReset: -1 are used for activeDescendant behaviour. These values can be abstracted away with
-// a new "type" option (roving or active)
-
 const defaultOptions = {
     axis: 'both',
-    autoInit: 0,
-    autoReset: null,
-    ignoreButtons: false,
+    autoInit: 'interactive',
+    autoReset: 'current',
+    ignoreByDelegateSelector: null,
     wrap: false
 };
 
-const itemFilter = (el) => !el.hidden;
+function findNavigableItems(items) {
+    return items.filter((el) => !el.hidden && el.getAttribute('aria-disabled') !== 'true');
```

Comment:
I think you should move some of the logic here into a separate function, there is a lot of repeated filtering that happens because you can't check the navigability of individual items elsewhere.
```js
function isNavigable(el) {
  return !el.hidden && el.getAttribute('aria-disabled') !== 'true'
}
```
```js
function findNavigableItems(items) {
    return items.filter(isNavigable);
}
```

Reply:
Good catch. I like that modularity/functional approach. Will change.